extensions/notebook/src/book/bookTreeItem.ts

```diff
@@ -22,6 +22,11 @@ export interface BookTreeItemFormat {
 	version?: BookVersion;
 	parent?: BookTreeItem;
 	hasChildren?: boolean;
+	/**
+	 * Use to identify the hierarchy of nested book tree items.
+	 * For instance, the hierarchyId of the first node would be "0" and its children would have
+	 * a hierarchyId starting with "0/"
```

Comment:
What node? And what about children nested beyond the first - are they going to look something like `0/0/1...`?

It would also be good to have some sort of explanation of what this would be used for in here as well.

Reply:
Yup, it would be a string lie '0/0/1...'